src/map/clif.c

```diff
@@ -16163,14 +16273,32 @@ void clif_parse_cz_config(int fd, struct map_session_data *sd)
 	int type = RFIFOL(fd, 2);
```

Comment:
this really not for this pr, but probably better fix too.
``int type`` should be ``enum CZ_CONFIG type``